src/index.tsx

```diff
@@ -28,12 +28,21 @@ const INITIAL_DATA = {
   }
 };
 
-function getPositionX() {
-  return window.pageXOffset || 0;
+function hasOwnProp(obj: Object, prop: string) {
+  if (typeof obj === 'undefined' || obj === null) return false;
+  return Object.hasOwnProperty.call(obj, prop);
```

Comment:
Shoot, I just realized I made a stupid mistake here. This check won't work. I'll fix it.